deployer/hub.py

```diff
@@ -294,10 +285,8 @@ def auth_azure(self):
                         "az",
                         "aks",
                         "get-credentials",
-                        "--name",
-                        cluster,
-                        "--resource-group",
-                        resource_group,
+                        f"--name={cluster}",
+                        f"--resource-group={resource_group}",
```

Comment:
Do you have `az` installed? If so, can you trial if it can parse flags like `--something=value`?

I've tried it works for kubectl / helm

Reply:
Yep, works for `az`